### PR TITLE
[Panda] PAN-3180: BpkPrice component update style

### PR DESCRIPTION
**Pull request by hchen-skyscanner**
[PAN-3180](https://skyscanner.atlassian.net/browse/PAN-3180)

This PR includes following changes: 
- Previous price text change to textError colour
- Previous price alignment change on `right` style

| Light | Dark | 
| -- | -- |
| light | dark |

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [ ] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack-SwiftUI/Price/Classes/BPKPrice.swift [hunk 1]
```diff
@@ -142,7 +143,7 @@ public struct BPKPrice: View {
             items.append(leadingText)
         }
         
-        return alignment == .trailing ? items.reversed() : items
```
Comment (frugoman) [inline Backpack-SwiftUI/Price/Classes/BPKPrice.swift:145]: Why is this changed? Was it on purpose?

Comment (frugoman) [reply Backpack-SwiftUI/Price/Classes/BPKPrice.swift:145]: Ah yes, seems okay

**Review by frugoman — APPROVED** [review]

File: Backpack-SwiftUI/Price/Classes/BPKPrice.swift (hunk 1 above)
Comment (hchen-skyscanner) [reply Backpack-SwiftUI/Price/Classes/BPKPrice.swift:145]: Yeah, in new design, the left and right alignmane will share the same order, so we remove the branch here
image
